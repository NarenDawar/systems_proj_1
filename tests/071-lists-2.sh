
true && true && true && echo true test success
false || false || false || echo false test success
