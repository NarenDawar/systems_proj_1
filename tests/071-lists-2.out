true test success
false test success
